script.js

```diff
@@ -77,25 +77,32 @@ init();
 window.addEventListener('resize', init);
 
 document.querySelector('.arrow-left').addEventListener('click', function () {
-  count++;
-  if (count >= images.length ) {
-      count = 0;
-  }
+  sliderLine.style.transition = "all ease .5s";
+  count <= 0 ? false : count--;
   rollSlider();
+  jumpSlider();
 });
 
 document.querySelector('.arrow-right').addEventListener('click', function () {
-  count--;
-  if (count < 0) {
-      count = images.length - 1;
-  }
+  sliderLine.style.transition = "all ease .5s";
+  count >= images.length - 1 ? false : count++;
   rollSlider();
+  jumpSlider();
 });
 
 function rollSlider() {
   sliderLine.style.transform = 'translate(-' + count * width + 'px)';
 }
 
+function jumpSlider() {
+  sliderLine.addEventListener('transitionend', function () {
+    count === 0 ? count = images.length - 2 : count;
```

Comment:
For such numbers in the code it is better to use constants with proper names. The reviewer doesn't have to  guess what is 2 or 3 for. Refactor it.

Reply:
DONE!